cost/azure/unused_ip_addresses/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## v3.3
 
+- remove duplicate data fields for subscriptionID and subscriptionName
+
+## 3.2
```

Comment:
As there was a merged pull request that affected this policy:

1. On Line 7 - change from `## 3.2` to `## v3.3`
2. On Line 3 - change from `## v3.3` to `## v3.4`